prometheus/src/main/java/org/opensearch/sql/prometheus/storage/PrometheusMetricScan.java

```diff
@@ -32,8 +33,9 @@ public class PrometheusMetricScan extends TableScanOperator {
 
   @EqualsAndHashCode.Include
   @Getter
+  @Setter
   @ToString.Include
-  private final PrometheusQueryRequest request;
+  private PrometheusQueryRequest request;
```

Comment:
Is it mutable?

Reply:
Yes....This request is set up by different types of commands.